src/main/java/org/jahia/modules/reports/bean/ReportContentMarkedForDeletion.java

```diff
@@ -116,6 +116,7 @@ public void addItem(JCRNodeWrapper node) throws RepositoryException {
         if (!node.isNodeType("jnt:page")) {
             itemParentPage = JCRContentUtils.getParentOfType(node, "jnt:page");
         }
+        getPublicationStatusOfANode(node);
```

Comment:
it's used for what?

Reply:
For a test, I forgot to remove this line.